ingest-app/resources/bulk_update_schema.json

```diff
@@ -12,6 +12,10 @@
         "UpdateTypeEnum": {
             "type": "string",
             "enum": ["ADD_TO_EXISTING", "CLEAR_FIELD", "CLEAR_ALL_AND_REPLACE", "FIND_AND_REMOVE", "FIND_AND_REPLACE"]
+        },
+        "UpdateFieldEnum": {
+          "type": "string",
+          "enum": ["SCIENCE_KEYWORDS", "LOCATION_KEYWORDS", "DATA_CENTERS", "PLATFORMS", "INSTRUMENTS"]
```

Comment:
I may have missed them, but I did not see tests for Location keywords changes. 

I think it makes sense to test every permutation of each Update type against each field type to be certain all of the bulk updates work (and continue to work when people make changes to UMM translation in the future).

I couldn't tell if you've done that.